### PR TITLE
Update page on python libraries

### PR DESCRIPTION
**Pull request by 8BitJosh**
Libraries synced between the image and simulator, python libs page updated to reflect this

Simulator libraries https://github.com/srobo/competition-simulator/blob/main/libraries.txt
Robot image libraries https://github.com/srobo/robot-image/blob/main/libraries.txt
Robot image requirements https://github.com/srobo/robot-image/blob/main/requirements.txt

### CODE REVIEW

File: kit/brain_board/python_libraries.md [hunk 1]
```diff
@@ -3,74 +3,47 @@ layout: page
...
+* [sr-robot3 2024.0.0](https://pypi.org/project/sr-robot3)
 <!-- cspell:enable -->
 
-If you wish to use a library that isn't listed above, get in contact with us on Discord and have a chat with us about it.
 
+## Simulator differences
 
-## Simulator
+Generally we will try to keep the libraries available in the simulator the same as what is available on the physical robot.
+The list below outlines the differences.
```
Comment (PeterJCLaw) [inline kit/brain_board/python_libraries.md:43]: Do we want to note here that the `sr-robot3` available in the simulator isn't the one from PyPI?

File: kit/brain_board/python_libraries.md [hunk 2]
```diff
@@ -3,74 +3,47 @@ layout: page
...
-
-<!-- updating this list? Also update https://github.com/srobo/competition-simulator/blob/master/libraries.txt -->
 <!-- cspell:disable -->
-
-* [matplotlib 3.5.1](https://pypi.org/project/matplotlib/3.5.1/)
-* [numpy 1.22.3](https://pypi.org/project/numpy/1.22.3/)
-* [pandas 1.4.2](https://pypi.org/project/pandas/1.4.2/)
-* [typing-extensions 3.10.0.0](https://pypi.org/project/typing-extensions/3.10.0.0/)
+* `flask` is not available in the simulator
+* `opencv` is not available in the simulator
```
Comment (PeterJCLaw) [inline kit/brain_board/python_libraries.md:47]: `debugpy` is also not available:
```suggestion
* `debugpy` is not available in the simulator
* `flask` is not available in the simulator
* `opencv` is not available in the simulator
```

File: kit/brain_board/python_libraries.md [hunk 3]
```diff
@@ -3,74 +3,47 @@ layout: page
 title: Available Python Libraries
 ---
 
+<!-- Simulator libraries https://github.com/srobo/competition-simulator/blob/main/libraries.txt -->
+<!-- Robot image libraries https://github.com/srobo/robot-image/blob/main/libraries.txt -->
+<!-- Robot image requirements https://github.com/srobo/robot-image/blob/main/requirements.txt -->
```
Comment (PeterJCLaw) [inline kit/brain_board/python_libraries.md:8]: These urls don't work; https://github.com/srobo/robot-image/blob/main/files/python/libraries.txt is what I've been referring to for the simulator.

File: kit/brain_board/python_libraries.md (hunk 2 above)
Comment (8BitJosh) [reply kit/brain_board/python_libraries.md:48]: Fixed c850300

File: kit/brain_board/python_libraries.md (hunk 3 above)
Comment (8BitJosh) [reply kit/brain_board/python_libraries.md]: Fixed c850300

File: kit/brain_board/python_libraries.md (hunk 1 above)
Comment (8BitJosh) [reply kit/brain_board/python_libraries.md:43]: Hows this c850300

**Review by PeterJCLaw — APPROVED** [review]
:+1:

One extra thought is whether we want to include the version numbers of packages which are also installed & likely to be useful, but which maybe aren't currently explicitly pulled in? I guess one answer might be that we should pull them in explicitly. (I'm thinking of `contourpy==1.1.0` and `kiwisolver==1.4.5`, which I pinned in the simulator to this end)

**Pull request merged**